src/ui-components/datatable/datatable.component.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+
+import { useTranslation } from "react-i18next";
+import { Link } from "react-router-dom";
+import {
+  Button,
+  DataTable,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+  TableToolbar,
+  TableToolbarContent
+} from "carbon-components-react";
+import { Add16 } from "@carbon/icons-react";
+import "./datatable.scss";
+
+const WidgetDataTable = ({
+  title,
+  rows,
+  headers,
+  linkTo,
```

Comment:
Link to what? Maybe this should be `seeAllLinkTarget` or something?

Reply:
Or `detailViewLink`, something like that

Reply:
Sure, `detailViewLink` sounds clearer.

Reply:
Ping on this one